.github/workflows/aws.yml

```
@@ -64,26 +63,43 @@ jobs:
       id: login-ecr
       uses: aws-actions/amazon-ecr-login@v1
 
+#    - name: Build, tag, and push image to Amazon ECR
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)